### ISSUE TITLE
I am investor, used this wallet, funds stolen

### ISSUE THREAD

**Issue by sunmother** [arancauchi/IOTA-Paper-Wallet#8]
I used this wallet twice for a small investment I made (the rest is still on the exchange).  The last time was on 11/23.  On 11/24, the balance was transferred out to an unknown address NCSL9R9YARCEOROHARFJEQDBCTELECYPKMJTABLTVGR9DWQMJZDKEGTASTIIJIPYWFNR9OZMTLCMGUHDCCOTXMANXD.  As investors, we want our investments to be safe.  Are there special precautions to be taken with this wallet?

**Comment by seabirdsea** [comment]
im am no tech guy but did you make your seed offline? if you used any online seed maker they have been problems and scams.

**Comment by arancauchi** [comment]
As seabirdsea said, this is overwhelmingly the most likely option. If you followed my instructions to use the paper wallet offline, there is simply no way my generator could have touched your seed at any point. This leaves the possibility that wherever you generated your seed was compromised, or you have malware on your computer.

**Comment by sunmother** [comment]
the paper wallet generated the seed.  I used it twice to transfer a balance on the exchange.  The paper wallet only has one address on it, though.  I am assuming that is the problem now that I have read all about double using addresses.  Sorry, I am an investor, not programmer, so I don't understand the ins and outs, but was told not to store anything on the exchanges back when I found your wallet.  I see there is a lite wallet now, but I am not sure how you change the address for the paper wallet, or if it was just intended for single use.
